### PR TITLE
 [13/N] Fix clang-tidy warnings in aten/src/ATen 

### PR DESCRIPTION
**Pull request by cyyever**
Follows #133425

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: aten/src/ATen/native/nested/NestedTensorUtils.h [hunk 1]
```diff
@@ -220,8 +219,8 @@ namespace impl {
 template <typename T>
 struct NestedNode {
   NestedNode() = delete;
-  explicit NestedNode(std::vector<T>&& children)
-      : _is_leaf(false), _children(children) {}
+  explicit NestedNode(std::vector<T> children)
```
Comment (Skylion007) [inline aten/src/ATen/native/nested/NestedTensorUtils.h:222]: I mean, yu could still leave it as a universal reference if you wanted...

**Review by Skylion007 — APPROVED** [review]

File: aten/src/ATen/native/nested/NestedTensorUtils.h (hunk 1 above)
Comment (cyyever) [reply aten/src/ATen/native/nested/NestedTensorUtils.h:222]: For universal reference it still requires a move operation later..

**Pull request closed without merging**